securedrop/source_app/utils.py

```diff
@@ -74,7 +75,10 @@ def wrapper(*args, **kwargs):
 
 
 @asynchronous
-def async_genkey(crypto_util_, db_uri, filesystem_id, codename):
+def async_genkey(crypto_util_,
```

Comment:
Do we have a type for this?

Reply:
that is the main object in `crypto_util.py` (`CryptoUtil`)